examples/browser-verify/README.md

```diff
@@ -24,5 +26,6 @@ where `$BROWSER` is one of
 - `firefox`
 - `safari`
 
+[install Rust]: https://doc.rust-lang.org/cargo/getting-started/installation.html
```

Comment:
This isn't used... did you want the intro to be more in line with the phrases of the other examples (which would use this link)?

Reply:
Ahh yes, good catch. I've removed the unused [install rust] link. 
The extra dependency of [Node.js] made the flow more natural to me with `install` outside of the link text.